openmetadata-service/src/main/java/org/openmetadata/service/util/FullyQualifiedName.java

```diff
@@ -68,6 +68,15 @@ public static String getParent(String fqn) {
     return parent;
   }
 
+  public static String getRoot(String fqn) {
+    // Split fqn of format a.b.c.d and return the root a
+    String[] split = split(fqn);
+    if (split.length <= 1) {
+      return null;
+    }
+    return split[0];
+  }
```

Comment:
We are not using it anywhere.

Reply:
Will keep it for now. I have plans of using this in a subsequent iteration.